tools/reviewer/local_server/service/tests/CodeReviewServiceTextDiffTest.java

```diff
@@ -266,7 +266,6 @@ public void testTextDiff_committedAndWorkspaceNotExists() {
             .build();
 
     TextDiffResponse response = getResponse(file);
```

Comment:
It isn't necessary to use the `response` variable.

Reply:
Correct :)